### PR TITLE
Updated docker file

### PR DESCRIPTION
**Pull request by MyDigitalLife**
Removed packages that are not needed anymore because there in the vendor.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,4 @@ FROM golang:1.8
 
 WORKDIR /go/src/
 
-RUN apt-get update && apt-get install -y libncurses5-dev
-RUN go get google.golang.org/api/compute/v1
-RUN go get cloud.google.com/go/compute/metadata
\ No newline at end of file
+RUN apt-get update && apt-get install -y libncurses5-dev
```
Comment (kris-nova) [inline docker/Dockerfile]: Do we still need this with #314 still?

Comment (xmudrii) [inline docker/Dockerfile]: Nope, it's not needed anymore.

Comment (MyDigitalLife) [inline docker/Dockerfile]: This was already removed in #314 so it should be gone when merged :)

**Comment by prateekgogia** [comment]
I am not sure, if we even need this Dockerfile.

**Comment by MyDigitalLife** [comment]
No it's not necessary at this point seeing the dependency was removed. It could be fully removed if we want to. But we could leave it there for a while seeing we might be adding dependencies in the future.

**Comment by prateekgogia** [comment]
@MyDigitalLife Can we please remove the file for now. We can always add it later, if we need to add dependencies.

**Comment by MyDigitalLife** [comment]
@prateekgogia Sure, I'll try and do this tonight.

**Pull request merged**